cli/endpoints/online/model-1/environment/conda.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - scikit-learn=0.24.2
   - scipy=1.7.1
   - pip:
-    - azureml-defaults==1.33.0
+    - itsdangerous<2.0,>=0.24
```

Comment:
Do we need the library itsdangerous? @spacedragon  @petrodeg

Reply:
Yes, we need to pin as it has broken all our runners. Klein has more details